.gitignore

```diff
@@ -12,3 +12,6 @@
 # For clangd, required to be here by hedronvision/bazel-compile-commands-extractor.
 # IDE files in general should go in .git/info/exclude.
 /.cache/
+
+# Editors
+*.swp
```

Comment:
Place in your local `.git/info/exclude`-file, in your global `.gitignore`, or configure vim to place the .swp-files somewhere better. I don't want people to start adding all their IDE- and editor-configuration to this file.

Reply:
I mean, that's what the .gitignore is for. But alright.